### ISSUE TITLE
bug: ion-datetime not working on devices with high accessibility settings

### ISSUE THREAD

**Issue by nathantaal** [ionic-team/ionic#21431]
# Bug Report

**Ionic version:**
[x] **4.x**

**Current behaviour:**
ion-datetime (for time) doesn't show value

**Expected behaviour:**
ion-datetime showing values

**Steps to reproduce:**
1. Open settings on an Samsung phone
2. Go accessibility -> vision -> screen zoom and set screen zoom and font size to the highest level possible.
3. Open the ion-datetime and don't see any values

**Related code:**
```
          <ion-datetime
                           :display-format="HH:mm"/>
```

**Other information:**
See example here: https://cp.sync.com/dl/8f6e239e0/epgb5j8g-drmkhr67-nivk26d8-uatmw25k

**Ionic info:**
```
Ionic:

   Ionic CLI       : 5.4.15
   Ionic Framework : @ionic/vue 0.0.9

Capacitor:

   Capacitor CLI   : 1.5.1
   @capacitor/core : 1.5.1

System:

   NodeJS : v12.18.0 (/usr/bin/node)
   npm    : 6.13.6
   OS     : Linux 5.4
```

**Comment by nathantaal** [comment]
**Prerequisite**
An Samsung device with high accessibility settings enabled (settings -> accessibility -> vision -> font and screen zoom)
1. Click the button 'larger font size'
2. Slide 'Screen Zoom' to Large
3. Slide 'Font Size' to extra huge
Works on any Samsung device that has the settings as described above. I tried it on S10 and J7 for this issue, but on several other devices before.

**Steps to reproduce**
This was happening to me on a Vue project as @liamdebeasi added the label, but I figured is happening inside the component. This example project is therefore written in React. 

1. Build from source or install app-debug.apk to a prepared (virtual) Samsung device
2. Try to change the time using the time input

**Extra info**
We can use https://github.com/phonegap/phonegap-mobile-accessibility to ignore the zoomlevel used by the Android user, it fixes text getting larger but doesn't not fix this particular issue (and the question is of course if we would even want that).

Checkout source code and install this 
https://github.com/nathantaal/ionic-timepicker-bug-example
